src/hooks.ts

```diff
@@ -0,0 +1,13 @@
+export type OnUnknownFieldName = (info: {
+  unknownFieldName: string
+  error: Error
+  validFieldNames: string[]
+  typeName: string
+}) => void
+
+export type OnUnknownOutputType = (info: {
```

Comment:
```suggestion
export type OnUnknownFieldType = (info: {
```

Might make more sense

Reply:
Other instances of this in the PR would need changing too then.